aggregators/rules/cr.js

```diff
@@ -1,16 +1,96 @@
+var request = require("request");
+var console = require("console");
+
+const CR_ADDRESS = process.env.CR_ADDRESS; // http://media.wizards.com/2016/docs/MagicCompRules_20160930.txt or newer
+
 class CR{
     constructor(){
-        this.Location = "http://blogs.magicjudges.org/rules/cr/";
+		this.location = "http://blogs.magicjudges.org/rules/cr/";
+		request({url: CR_ADDRESS, encoding: "utf16le"}, (error, response, body) => {
+			 if (!error && response.statusCode == 200) {
+				 let {cr, glossary} = parseCR(body);
+				 this.cr = cr;
+				 this.glossary = glossary;
+				 console.log("CR Ready")
+			 } else {
+				 console.error(error);
+			 }
+		});
     }
     find (parameter,callback){
```

Comment:
Imho this belongs in the constructor, we want to load and parse the CR exactly once. We could do some delayed initialization in getContent, but I don't see the need.